src/js/helpers/constants.js

```diff
@@ -0,0 +1,17 @@
+export const CONSTANTS = {
+  API_KEY: '743c48cccf844d00914aa7c2e83f0e31',
+  SERVICE_URL_SOURCES: 'https://newsapi.org/v2/sources',
+  SERVICE_URL_ARTICLES: 'https://newsapi.org/v2/top-headlines',
+
+  COMMON_ERROR: 'An error occurred during getting information!',
+  NOTEXIST_ERROR: 'Current source does not exist!',
+
+  ARTICLES: 'Articles'
+};
+
+export const ELEMENTS = {
```

Comment:
The purpose of this hometask is use a more as possible pattern.
It would be better use `FactoryMethod` than constant.
You can take a look in patch file that you can find into common comment.

Reply:
done